components/bl-webcamPhoto-component/src/utils/data-url-to-blob.js

```diff
@@ -0,0 +1,4 @@
+export async function dataURLToBlob(dataURL) {
```

Comment:
dataURLToBlob
toBase64

into one file 'utils/file.js'

other to 'utils/device.js'